modules/ui/src/main/scala/lucuma/ui/components/state/ConnectionManager.scala

```diff
@@ -25,7 +26,9 @@ case class ConnectionManager(
 )(using
   val logger:       Logger[DefaultA]
 ) extends ReactFnPropsWithChildren(ConnectionManager.component):
-  val payload: Map[String, Json] = Map("Authorization" -> vault.authorizationHeader.asJson)
+  val payload: Map[String, Json] = Map(
+    Authorization.name.toString -> vault.authorizationHeader.credentials.renderString.asJson
```

Comment:
Why  is this better than the  plain string?

Reply:
In some places we want to pass this to http4s, which requires a structured header. It's more straightforward to provide a structure here and convert it to string when necessary than split a string to build a structure.